presto-main/src/main/java/com/facebook/presto/server/protocol/Query.java

```diff
@@ -559,8 +499,9 @@ private synchronized URI createNextResultsUri(String scheme, UriInfo uriInfo)
     {
         return uriInfo.getBaseUriBuilder()
                 .scheme(scheme)
-                .replacePath("/v1/statement")
+                .replacePath("/v1/statement/executing")
```

Comment:
I'm assuming you've talked to @martint or @electrum offline about the specific API change. I know we had a general discussion previously.